ci/deploy.sh

```diff
@@ -22,14 +22,19 @@ git checkout "$TRAVIS_BRANCH"
 git remote set-url origin "git@github.com:$TRAVIS_REPO_SLUG.git"
 
 # Decrypt and add SSH key
+eval "$(ssh-agent -s)"
+if [ -v MANUBOT_SSH_PRIVATE_KEY ]; then
+  set +o xtrace  # TODO: better way of temporarily disabling xtrace
+  base64 --decode <<< "$MANUBOT_SSH_PRIVATE_KEY" | ssh-add -
+else
```

Comment:
Previously we didn't disable xtrace, which means we probably were leaking `$encrypted_9befd6eddffe_key` to the build logs. However, if you check the build logs, I [believe](https://docs.travis-ci.com/user/best-practices-security/#steps-travis-ci-takes-to-secure-your-data) Travis CI was smart enough to remove that output.

Anyways we want a way to disable xtrace for specific commands. Ideally, after these commands finish, xtrace operates based on whether it was activate or not before the hidden commands.

@michaelmhoffman / @dongbohu any suggestions? Would also love any feedback on the changes to the bash command changes above since would be great to make this as robust as possible.

Reply:
Implemented xtrace disabling/re-enabling in https://github.com/manubot/rootstock/pull/243/commits/c1abee3d0c83ddbea5a3eea28e846e1ff6a5928d.

Reply:
Again I don't recommend using `xtrace` routinely like this. If you must, then you can disable it temporarily in a subshell like this:

```
(
    set +o xtrace
    base64 --decode <<< "$MANUBOT_SSH_PRIVATE_KEY" | ssh-add -
)
```

Remember that variables changed in the subshell don't affect the parent shell. That is not a problem now here.